lcls_tools/common/devices/yaml/DL1.yaml

```diff
@@ -606,10 +606,10 @@ screens:
   OTR3:
     controls_information:
       PVs:
-        image: OTRS:IN20:621:Image:ArrayData
+        image: OTRS:IN20:621:IMAGE
         n_bits: OTRS:IN20:621:N_OF_BITS
```

Comment:
OTR2 and 3 seemed to switch camera PVs here? Did you update/ double check this?

Reply:
Can you pleases write a bug issue on this and reference the code lines, etc. Thanks in advance.